### PR TITLE
Fixes a couple doc bugs for UserCredential and UserAssertion

### PR DESCRIPTION
**Pull request by psignoret**

### CODE REVIEW

**Comment by psignoret** [comment]
.

File: src/ADAL.NET/UserCredential.cs [hunk 1]
```diff
@@ -35,7 +35,7 @@ public UserCredential(string userName, string password)
         }
 
         /// <summary>
-        /// Constructor to create credential with client id and secret
+        /// Constructor to create credential with with username and password
```
Comment (kpanwar) [inline src/ADAL.NET/UserCredential.cs]: double with.

**Comment by psignoret** [comment]
Fixed, thanks.

**Comment by psignoret** [comment]
Fixed, thanks.

**Pull request merged**